package.json

```diff
@@ -61,7 +62,9 @@
     "ldapjs": "1.0.0"
   },
   "devDependencies": {
+    "autoprefixer": "6.4.1",
     "chai": "3.5.0",
+    "cssnano": "3.7.5",
```

Comment:
Did you notice any performance improvement? I'd personally prefer not minify that, unless it really is a problem.